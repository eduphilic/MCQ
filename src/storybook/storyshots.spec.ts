import initStoryshots from "@storybook/addon-storyshots";
import render from "../test/renderer";

initStoryshots({
	configPath: __dirname,
	renderer: render,
});
