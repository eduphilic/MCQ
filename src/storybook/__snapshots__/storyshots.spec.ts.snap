// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots StickyFooter default 1`] = `
<div
  className="StickyFooter-pddbet-0 StickyFooter-pddbet-0"
>
  StickyFooter
  <footer
    className="StickyFooter__StickyFooterText-pddbet-1 bDhSqg"
  >
    <span
      className="MuiTypography-root-1 MuiTypography-caption-4 MuiTypography-colorTextPrimary-26"
    >
      Copyright : Eduphilic Consultancy Pvt Ltd 
      2019
    </span>
  </footer>
</div>
`;
